script.js

```diff
@@ -1,4 +1,146 @@
+// Global variables
+let currentPlayer = 1;
+let numOfPlayers = 3;
+var numOfDice = 2;
+
+// declare 2 game modes so that the submit button will call main function 2 different ways
+var currGameMode = "roll dice"; // second game mode will be "choose order". Use if statements for this!
+
+var dice1 = "";
+var dice2 = "";
+
+var playerNumbers = [];
+var winnersArr = [];
+
+// Helper Functions
+var generateRandomNumber = function () {
+  var dicevalue = Math.floor(Math.random() * 6) + 1;
+  return dicevalue.toString(); // to combine the values via two options. Number data types don't work like that
+};
+
+var areDicesSame = function () {
```

Comment:
whenever you see yourself return true or false, check if you can just return the expression directly. i.e. 
`return dice1 == dice2`